knack/index.css

```diff
@@ -1,31 +1,17 @@
 /* hide attachment type column (field contents are extracted via JS */
-div.view_2107 .field_2403 {
-  display: none;
-}
-div.view_2108 .field_2403 {
-  display: none;
-}
-div.view_2565 .field_2403 {
-  display: none;
-}
+div.view_2107 .field_2403 { display: none; }
```

Comment:
Any reason you changed the formatting here? We're trying to get into the habit of using [Prettier](https://prettier.io/) for auto-formatting with CSS & JS files and installed in our code editors. You've rolled some of that back with this commit. You might have missed that discussion in our mid-sprint dev sync this week.

Reply:
thanks for flagging that. it would seem that the css in the actual Knack app was not prettified. i'll get prettier working and open a new PR to resolved formatting.

Reply:
ah, that would make sense. My bad.